hudi-common/src/main/java/org/apache/hudi/common/util/JsonUtils.java

```diff
@@ -60,7 +62,7 @@ public static String toString(Object value) {
   }
 
   public static void registerModules() {
-    // NOTE: Registering [[JavaTimeModule]] is required for Jackson >= 2.11 (Spark >= 3.3)
+    // NOTE: Registering [[JavaTimeModule]] is required for Jackson >= 2.11 (Spark >= 3.2)
```

Comment:
So after this change, spark 3.2 also needs to register the JSON modules explicitly, can you elaborate a little more about this change?

Reply:
FYI, Hudi is going to deprecate the support for older Spark versions (Spark 2, 3.0, 3.1, 3.2) soon in Hudi 1.0 (#11523).

Reply:
@danny0405 
The part that needed to be modified was just [adding the spark 3.2 jackson version](https://github.com/apache/hudi/pull/11548/commits/0528626988810f9ab31105bb739c8e97d676a755), but I corrected the related part by correcting the failed test.
This part appears to only cover spark version 3.3 or higher, so it was changed to include version 3.2.
In fact, BaseSpark3Adapter use this, and only 3.2Adapter has implemented it.
If version 3.2 is scheduled to be deprecated, it may be something that can be removed.